apps/projects/app/components/Shared/FilterBar/prepareFilters.js

```diff
@@ -66,6 +67,7 @@ const prepareFilters = (issues, bountyIssues) => {
       } else {
         filters.labels['labelless'] = {
           name: 'Issues without labels',
+          color: 'F2F2F2',
```

Comment:
Should these values come from `theme`?

Reply:
I wasn't sure what it should be - this color is for Tags that are artificially created for labelless issues. As suggested on #design I changed that to a) not include color at all, which b) signifies that an outline is needed instead (in FilterBar itself).